bot/seasons/easter/easter_riddle.py

```diff
@@ -0,0 +1,85 @@
+import asyncio
+import logging
+import random
+from json import load
+from pathlib import Path
+
+import discord
+from discord.ext import commands
+
+from bot.constants import Colours
+
+log = logging.getLogger(__name__)
+
+with open(Path('bot', 'resources', 'easter', 'easter_riddle.json'), 'r', encoding="utf8") as f:
+    RIDDLE_QUESTIONS = load(f)
+
+TIMELIMIT = 10
+
+
+class EasterRiddle(commands.Cog):
+    """This cog contains the command for the Easter quiz!"""
+
+    def __init__(self, bot):
+        self.bot = bot
+        self.quiz_messages = {}
+        self.winner = ""
+        self.correct = ""
+
+    @commands.command(aliases=["riddlemethis", "riddleme"])
+    async def riddle(self, ctx):
+        """Gives a random riddle questions, then provides 2 hints at 10 second intervals before revealing the answer"""
+        random_question = random.choice(RIDDLE_QUESTIONS)
+        question = random_question["question"]
+        hints = random_question["riddles"]
+        self.correct = random_question["correct_answer"]
+
+        description = f"You have {TIMELIMIT} seconds before the first hint.\n\n"
+
+        q_embed = discord.Embed(title=question, description=description, colour=Colours.pink)
+
+        await ctx.send(embed=q_embed)
+        await asyncio.sleep(TIMELIMIT)
+
+        h_embed = discord.Embed(
+            title=f"Here's a hint: {hints[0]}!",
+            colour=Colours.pink
+        )
+
+        await ctx.send(embed=h_embed)
+        await asyncio.sleep(TIMELIMIT)
+
+        h_embed = discord.Embed(
+            title=f"Here's a hint: {hints[1]}!",
+            colour=Colours.pink
+        )
+
+        await ctx.send(embed=h_embed)
+        await asyncio.sleep(TIMELIMIT)
+
+        if not self.winner:
```

Comment:
Shouldn't this be reversed? `if not self.winner` will be `True` when there aren't any winners.